plugins/woocommerce/src/Admin/ReportCSVEmail.php

```diff
@@ -50,17 +50,30 @@ public function __construct() {
 		$this->template_base  = WC()->plugin_path() . '/includes/react-admin/emails/';
 		$this->template_html  = 'html-admin-report-export-download.php';
 		$this->template_plain = 'plain-admin-report-export-download.php';
-		$this->report_labels  = array(
-			'categories' => __( 'Categories', 'woocommerce' ),
-			'coupons'    => __( 'Coupons', 'woocommerce' ),
-			'customers'  => __( 'Customers', 'woocommerce' ),
-			'downloads'  => __( 'Downloads', 'woocommerce' ),
-			'orders'     => __( 'Orders', 'woocommerce' ),
-			'products'   => __( 'Products', 'woocommerce' ),
-			'revenue'    => __( 'Revenue', 'woocommerce' ),
-			'stock'      => __( 'Stock', 'woocommerce' ),
-			'taxes'      => __( 'Taxes', 'woocommerce' ),
-			'variations' => __( 'Variations', 'woocommerce' ),
+
+		/**
+		 * Used to customise report email labels.
+		 *
+		 * @since x.x.x
```

Comment:
```suggestion
		 * @since 9.9.0
```

I assume this PR targets 9.9.0

Reply:
I updated it to 9.9.0. Thank you, @rawdreeg !